File: MobilneSystemyInformatyczne/ShareWindsurfingEquipment/src/main/java/com/project/ShareWindsurfingEquipment/model/UserReview.java
package com.project.ShareWindsurfingEquipment.model;

import javax.persistence.*;
import java.math.BigDecimal;

@Entity
@Table
public class UserReview {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String content;

    @Column
    private int rating;

    @Column
    private String reviewerLogin;

    @Column
    private String postReviewLogin;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public int getRating() {
        return rating;
    }

    public void setRating(int rating) {
        this.rating = rating;
    }

    public String getReviewerLogin() {
        return reviewerLogin;
    }

    public void setReviewerLogin(String reviewerLogin) {
        this.reviewerLogin = reviewerLogin;
    }

    public String getPostReviewLogin() {
        return postReviewLogin;
    }

    public void setPostReviewLogin(String postReviewLogin) {
        this.postReviewLogin = postReviewLogin;
    }

    @Override
    public String toString() {
        return "UserReview{" +
                "id=" + id +
                ", content='" + content + '\'' +
                ", rating=" + rating +
                ", reviewerLogin='" + reviewerLogin + '\'' +
                ", postReviewLogin='" + postReviewLogin + '\'' +
                '}';
    }
}

File: MSI-project/src/app/models/userReview.ts
export class UserReview {

  constructor(
    private _id: number,
    private _content: string,
    private _rating: number,
    private _reviewerLogin: string,
    private _postReviewLogin: string
  ){}

  public get content(): string {
    return this._content;
  }
  public set content(value: string) {
    this._content = value;
  }
  public get rating(): number {
    return this._rating;
  }
  public set rating(value: number) {
    this._rating = value;
  }
  public get reviewerLogin(): string {
    return this._reviewerLogin;
  }
  public set reviewerLogin(value: string) {
    this._reviewerLogin = value;
  }
  public get postReviewLogin(): string {
    return this._postReviewLogin;
  }
  public set postReviewLogin(value: string) {
    this._postReviewLogin = value;
  }
  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }
}
